components/fuipwahef749384h2e.yml

```diff
@@ -0,0 +1,23 @@
+name: emotional_bot_response
+display_name: Emotional Bot Response
+component_type: Generative
+model_type: NN-based
+is_customizable: true
+author: publisher@deeppavlov.ai
+description: Propose alternative hypotheses with specified emotional color.
+ram_usage: 500M
+gpu_usage: null
+group: candidate_annotators
+connector:
+  protocol: http
+  timeout: 15.0
```

Comment:
why this timeout is 15 seconds if your generative timeout is 120? fix it. Including in pipeline and othe config files. Maybe you can decrease the generative timeout (it won't take 120 secods because the prompt is yours hard-coded, so it would be a fast response by openai)

Reply:
changed generative timeout to 30 sec everywhere